mne/tests/test_ica.py

```diff
@@ -0,0 +1,77 @@
+from nose.tools import assert_true
+from unittest import TestCase
+
+import os.path as op
+from mne import Epochs, pick_types, read_events
+from mne.preprocessing import ICA, read_ica
+from mne.io import read_raw_fif
+from mne.datasets import sample
+from mne.utils import _TempDir, run_tests_if_main
+
+import warnings
+warnings.simplefilter('always')
+
+
+class TestLoadICA(TestCase):
```

Comment:
These tests should go in `mne/preprocessing/ica/tests/test_ica.py`

And we use functions for simplicity instead of classes. You'll also want to use the `requires_sklearn` decorator